d3/query-graphs.js

```diff
@@ -4,8 +4,13 @@ var path = require('path');
 
 // Require local modules
 var common = require('./common');
-var tableau = require('./tableau');
-var hyper = require('./hyper');
+var knownLoaders = {
```

Comment:
We currently bundle all scripts together and only ship one combined JS file.
I am not sure if it would actually improve performance, if we would load the loaders lazily.
With lazy-loading, the new flow would be:
* Download the main JS file
* Parse it and execute it
* Decide which loader to use
* Download the loader's JS file; parse it & execute it

With lazy-loading, we would introduce another roundtrip. Even worse, the browser can not start fetching the loader's JS file until after it executed the first JS file. During that time, we could probably download a few hundred additional KB